docs/relational-databases/security/encryption/setup-steps-for-extensible-key-management-using-the-azure-key-vault.md

```diff
@@ -21,6 +21,8 @@ ms.author: arupp
 
 [!INCLUDE [SQL Server](../../../includes/applies-to-version/sqlserver.md)]
 
+**Note:** SQL Server TDE Extensible Key Management by using Azure Key Vault is currently unsupported for SQL Server on Linux.
```

Comment:
We should probably re-phrase this as :
Extensible Key Management is not supported for SQL Server on Linux as documented here: https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-editions-and-components-2019?view=sql-server-ver16#Unsupported. Thus, this article does not apply for SQL Server on Linux.